nautobot/ipam/models.py

```diff
@@ -116,29 +117,43 @@ class VRF(PrimaryModel):
         through="ipam.VRFDeviceAssignment",
         through_fields=("vrf", "device"),
     )
-    prefixes = models.ManyToManyField(
-        to="ipam.Prefix",
+    virtual_machines = models.ManyToManyField(
+        to="virtualization.VirtualMachine",
         related_name="vrfs",
-        through="ipam.VRFPrefixAssignment",
+        through="ipam.VRFDeviceAssignment",
+        through_fields=("vrf", "virtual_machine"),
     )
+    # prefixes = models.ManyToManyField(
+    #     to="ipam.Prefix",
+    #     related_name="vrfs",
+    #     through="ipam.VRFPrefixAssignment",
+    # )
     tenant = models.ForeignKey(
         to="tenancy.Tenant",
         on_delete=models.PROTECT,
         related_name="vrfs",
         blank=True,
         null=True,
     )
+    enforce_unique = models.BooleanField(
+        default=True,
+        verbose_name="Enforce unique space",
+        help_text="Prevent duplicate prefixes/IP addresses within this VRF",
+    )
```

Comment:
This is gonna go away still right?

Reply:
Yes. I just needed it there to write the data migrations.